lib/pages/footer/SimpleToolTip.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class SimpleToolTip extends StatelessWidget {
```

Comment:
This widget should be named better, if it's a reusable widget then maybe name this `AppTooltip`.

Reply:
done